### PR TITLE
docs: add information on --windows flag and rewrite other bits of index.md

### PR DESCRIPTION
**Pull request by pfps**
I forgot to add this commit to PR #666

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -3,27 +3,41 @@ title: Solaar
 layout: default
 ---
 
-**Solaar** is a Linux manager for Logitech's peripherals that connect via a USB 
-[Unifying][unifying], Lightspeed, or Nano receiver.  It is able to pair/unpair devices with 
-receivers and for most devices show battery status.  Solaar can also control
-some of the changeable features of the devices, such as smooth scrolling or
-function key behavior.  Solaar does not work with Logitech peripherals that
+**Solaar** is a Linux manager for Logitech's keyboards and mice that connect via a USB
```
Comment (FFY00) [inline docs/index.md]: > keyboards and mice

`devices`

Comment (pfps) [reply docs/index.md]: Done

File: docs/index.md [hunk 2]
```diff
@@ -3,27 +3,41 @@ title: Solaar
...
 
 Solaar has progressed past version 1.0.  Problems with earlier versions should
 not be reported as bugs.  Instead upgrade to a recent version or manually install
 the current version from [GitHub](https://github.com/pwr-Solaar/Solaar).
+Some of the capabilities of Solaar have been developed by observing the behavior of
+Logitech receivers and devices and generalizing from these observations.
+If your Logitech receiver or device behaves in a strange way this may be caused by
+an incorrect behavior generalization.
+Please report such experiences by creating an issue in 
+[the Solaar repository](https://.../...Solaar...).
```
[the remark this answers is not in the document]
Comment (FFY00) [reply docs/index.md]: We should hardcode the url here.

[the remark this answers is not in the document]
Comment (pfps) [reply docs/index.md]: Yeah, stupid of me.  Fix coming.

[the remark this answers is not in the document]
Comment (pfps) [reply docs/index.md]: PR #690

**Pull request merged**